components/DropdownLight/DropdownLight.jsx

```diff
@@ -236,56 +306,132 @@ const DropdownLight = ({ disabled, items, theme, placeholder, name }) => {
   }, [enterPress]);
 
   return (
-    <Wrapper>
-      <input
-        type="text"
-        hidden
-        name={name}
-        defaultValue={selectedItem}
-        aria-label="selecione uma opção"
-      />
-
-      <Button
-        aria-haspopup="true"
-        aria-label={isOpen ? 'fechar lista de itens' : 'abrir lista de itens'}
-        onClick={handleToggleDropdown}
-        theme={theme}
-        disabled={disabled}
-        ref={buttonRef}
-      >
-        {itemLabel}
-        <ArrowIcon
-          name={isOpen ? 'arrow_drop_up' : 'arrow_drop_down'}
-          theme={theme}
-        />
-      </Button>
-
-      {isOpen && (
-        <SelectionList theme={theme} ref={listOptions}>
-          {items.map((item, index) => (
-            <SelectionListItem
-              role="option"
-              theme={theme}
-              key={item?.value || item}
-              onClick={() => handleOnClickListItem(item)}
-              aria-posinset={index}
-              aria-selected={index === cursor}
-              tabIndex="-1"
-            >
-              {item?.label || item}
-
-              {(selectedItem === item?.value || selectedItem === item) && (
-                <CheckIcon theme={theme} />
-              )}
-            </SelectionListItem>
-          ))}
-        </SelectionList>
-      )}
-    </Wrapper>
+    // <<<<<<< HEAD
+    <>
+      <ComponentWrapper theme={theme} skin={skin}>
+        <InputWrapper ref={wrapperRef}>
+          <InputLabel htmlFor={id} error={error}>
+            {label}
+            {required && <RequiredMark skin={skin}>*</RequiredMark>}
+          </InputLabel>
+          <InputText
+            type="text"
+            hidden
+            error={error}
+            skin={skin}
+            name={name}
+            placeholder={placeholder}
+            defaultValue={selectedItem}
+            aria-label="selecione uma opção"
+            htmlFor={id}
+            required={required}
+          />
+          <Button
+            aria-haspopup="true"
+            aria-label={
+              isOpen ? 'fechar lista de itens' : 'abrir lista de itens'
+            }
+            onClick={handleToggleDropdown}
+            theme={theme}
+            skin={skin}
+            error={error}
+            disabled={disabled}
+            ref={buttonRef}
+          >
+            {itemLabel}
+            {error ? (
+              <InputErrorIcon description={error} theme={theme} skin={skin} />
+            ) : (
+              <ArrowIcon
+                name={isOpen ? 'arrow_drop_up' : 'arrow_drop_down'}
+                theme={theme}
```

Comment:
Why do you use the "theme" attribute here on the icon?

Reply:
I was testing some variations and forgot to remove